pkgs/data/fonts/hubot-sans/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub }:
```

If there is no C compilation, use stdenvNoCC:

```suggestion
{ lib
, stdenvNoCC
, fetchFromGitHub 
}:
```

---

Very good, I didn't know that thanks :)